models/graphql/shopify_gql__orders.sql

```diff
@@ -0,0 +1,155 @@
+    from {{ ref('shopify_gql__orders__order_refunds') }}
+
+), refund_aggregates as (
+    select
+        order_id,
+        source_relation,
+        sum(subtotal) as refund_subtotal,
+        sum(total_tax) as refund_total_tax
+    from refunds
+    group by 1,2
+
+), order_discount_code as (
+    
+    select *
+    from {{ ref('int_shopify_gql__order_discount_code') }}
+
+), discount_aggregates as (
+
+    select 
+        order_id,
+        source_relation,
+        sum(case when type = 'shipping' then value_amount else 0 end) as shipping_discount_amount,
+        sum(case when type = 'percentage' then value_amount else 0 end) as percentage_calc_discount_amount,
+        sum(case when type = 'fixed_amount' then value_amount else 0 end) as fixed_amount_discount_amount,
+        count(distinct code) as count_discount_codes_applied
+
+    from order_discount_code
+    group by 1,2
+
+), order_tag as (
+
+    select
+        order_id,
+        source_relation,
+        {{ fivetran_utils.string_agg("distinct cast(value as " ~ dbt.type_string() ~ ")", "', '") }} as order_tags
+    
+    from {{ var('shopify_gql_order_tag') }}
+    group by 1,2
+
+), joined as (
+
+    select
+        orders.*,
+        {# QUESTION: in REST, we needed to parse a json to get this, but in gql it's already included in orders.*
+        but it has a slightly different name/is split out - should we include the below as well? #}
+        coalesce(shipping_cost_shop_amount, 0) as shipping_cost,
+        
+        order_adjustments_aggregates.order_adjustment_amount,
+        order_adjustments_aggregates.order_adjustment_tax_amount,
+
+        refund_aggregates.refund_subtotal,
+        refund_aggregates.refund_total_tax,
+
+        (orders.total_price_shop_amount
+            + coalesce(order_adjustments_aggregates.order_adjustment_amount,0) + coalesce(order_adjustments_aggregates.order_adjustment_tax_amount,0) 
+            - coalesce(refund_aggregates.refund_subtotal,0) - coalesce(refund_aggregates.refund_total_tax,0)) as order_adjusted_total,
+        order_lines.line_item_count,
+        order_lines.total_line_items_price_pres_amount,
+        order_lines.total_line_items_price_shop_amount,
+        order_lines.total_line_items_price_pres_currency_codes,
+        order_lines.total_line_items_price_shop_currency_codes,
+
+        coalesce(discount_aggregates.shipping_discount_amount, 0) as shipping_discount_amount,
+        coalesce(discount_aggregates.percentage_calc_discount_amount, 0) as percentage_calc_discount_amount,
+        coalesce(discount_aggregates.fixed_amount_discount_amount, 0) as fixed_amount_discount_amount,
+        coalesce(discount_aggregates.count_discount_codes_applied, 0) as count_discount_codes_applied,
+        coalesce(order_lines.order_total_shipping_tax, 0) as order_total_shipping_tax,
+        order_tag.order_tags,
+        {# QUESTION: order_url_tag was removed so no order_url_tag.order_url_tags -- should we set a null field? #}
```

Comment:
If the fields not included then I would say we should not include it from this version